docs/api/read/events.md

```diff
@@ -238,6 +238,21 @@ A `Calendar` representation:
         { "name": "endAfter", "type": "date", "class": "https://example.org/param/date-query" },
         { "name": "summary", "type": "text", "class": "https://example.org/param/free-text-query" }
       ]
+    },
+    {
+      "name": "Import",
+      "summary": "Import URL",
+      "method": "GET",
+      "href": "/v0/import/courses/1/classes/1920v/calendar{?type,startBefore,startAfter,endBefore,endAfter,summary}",
+      "isTemplated": true,
+      "fields": [
+        { "name": "type", "type": "text", "class": "https://example.org/param/free-text-query" },
```

Comment:
I wasn't sure if I should include the fields, but the action still supports the filtering therefore I left it.